simtools/ray_tracing.py

```diff
@@ -350,6 +355,56 @@ def _isValidKeyToPlot(self, key):
         '''
         return key in ['d80_cm', 'd80_deg', 'eff_area', 'eff_flen']
 
+    def plotAndSave(self, key, **kwargs):
+        '''
+        Plot key vs off-axis angle and save the figure in pdf.
+
+        Parameters
+        ----------
+        key: str
+            d80_cm, d80_deg, eff_area or eff_flen
+        **kwargs:
+            kwargs for plt.plot
+
+        Raises
+        ------
+        KeyError
+            If key is not among the valid options.
+        '''
+        if not self._isValidKeyToPlot(key):
+            msg = 'Invalid key to plot'
+            self._logger.error(msg)
+            raise KeyError(msg)
+
+        ylabel = {
```

Comment:
There is duplication between this list and the list in the _isValidKeyToPlot. I would simply make the dictionary of ylables a class variable and replace the function _isValidKeyToPlot with a simple `if not in ylabel.keys()`.

Reply:
Good idea.